src/outputs/history.cpp

```diff
@@ -179,8 +179,11 @@ void HistoryOutput::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm,
       PARTHENON_FAIL(msg);
     }
 
-    // If this is the first output, write header
-    if (output_params.file_number == 0) {
+    // Write the header lines once per invocation of Parthenon -- output variables may
```

Comment:
This pattern results in the header info only being written for the first history output, doesn't it?

Reply:
Not sure what you mean...  The old pattern wrote it once per simulation, only at output 0, so any restarts would not re-write (since output number was re-read at restart to be nonzero).  The new invocation has a static variable which is not stored persistently.

Reply:
OH now I understand sorry, I completely forgot about that case.  I'll just switch to a static `vector<bool>` for each output.